src/inference/dev_api/openvino/runtime/iplugin.hpp

```diff
@@ -184,6 +185,28 @@ class OPENVINO_RUNTIME_API IPlugin : public std::enable_shared_from_this<IPlugin
     virtual std::shared_ptr<ov::ICompiledModel> import_model(std::istream& model,
                                                              const ov::SoPtr<ov::IRemoteContext>& context,
                                                              const ov::AnyMap& properties) const = 0;
+    /**
+     * @brief Creates an compiled model from an previously exported model file using plugin implementation
+     *        and removes OpenVINO Runtime magic and plugin name
+     * @param model_buffer Reference to model memory object
+     * @param properties A ov::AnyMap of properties
+     * @return An Compiled model
+     */
+    virtual std::shared_ptr<ov::ICompiledModel> import_model(std::shared_ptr<ov::MappedMemory>& model_buffer,
```

Comment:
It is new API for Plugin, should implement it for all plugins: GPU/AUTO/BATCH/NPU?
or can we reuse the original `import_model(std::istream& model,..)` and encapsulate `ov::MappedMemory` into `std::istream`?

Reply:
Done via `Any` object